assembl/graphql/schema.py

```diff
@@ -511,6 +515,13 @@ class QuestionInput(graphene.InputObjectType):
     title_entries = graphene.List(LangStringEntryInput, required=True)
 
 
+# How the file upload works
+# With the https://github.com/jaydenseric/apollo-upload-client
```

Comment:
It isn't a folder on the server, I reuse the existing infrastructure. I create a Document object that contains a blob field, and I create an IdeaAttachment object between the document (image) and the thematic. I now remove the attachment objet AND the document (image) object when we update the image. There is no size limit. I don't think there is one in the old interface either.